app/views/notifications/index.html.erb

```diff
@@ -1,4 +1,7 @@
+<%= title("Notifcations") %>
```

Comment:
Unable to login in the review app to see the changes live. I get this:
![image](https://user-images.githubusercontent.com/85733202/159268773-5e131503-ddf2-4656-b793-62ae1b5bd98b.png)


But here's a typo:
```suggestion
<%= title("Notifications") %>
```